plugins/restlet1x/nexus-restlet1x-plugin/src/main/java/org/sonatype/nexus/rest/repositorystatuses/RepositoryStatusesListPlexusResource.java

```diff
@@ -76,6 +84,15 @@ public PathProtectionDescriptor getResourceProtection() {
   public Object get(Context context, Request request, Response response, Variant variant)
       throws ResourceException
   {
+    if (adminOnly) {
+      Subject subject = ThreadContext.getSubject();
+      // skip building reply unless user has admin access (ie. can update status)
+      if (subject == null || !subject.isPermitted("nexus:repostatus:update")) {
```

Comment:
sure - just pointing out that since the "nexus:repostatus:update" is not allocated to any of the out-of-the-box roles (it's only granted to nx-admin by virtue of the "nexus:*" wildcard) then either those users have an active need to update the status, in which case hiding the status makes that job harder - or they don't really need to update the status, in which case it's easy to remove that privilege from them and reduce the load.